frontend/src/scenes/insights/Insight.tsx

```diff
@@ -47,6 +52,18 @@ export function Insight({ insightId }: InsightSceneProps): JSX.Element {
             <div className="Insight">
                 <InsightPageHeader insightLogicProps={insightProps} />
 
+                {isObject(filtersOverride) && (
+                    <LemonBanner type="warning" className="mb-4">
+                        <div className="flex flex-row items-center justify-between gap-2">
+                            <span>You are viewing this insight with filters from a dashboard</span>
+
+                            <LemonButton type="secondary" to={urls.insightView(insightId as InsightShortId)}>
+                                Remove filters
```

Comment:
Possibly more clear that this is only about dashboard filters:
```suggestion
                                Discard dashboard filters
```